### ISSUE TITLE
pkg-catalogs issues with `pkg-install-command`

### ISSUE THREAD

**Issue by xgqt** [racket/racket#4427]
# 1. Setting `current-pkg-catalogs` parameter does not change the catalogs used by `pkg-install-command`
```
(require net/url)
(current-pkg-catalogs (list (string->url "https://google.com")))
(pkg-install-command "asd")
```
Result:
```
Resolving "asd" via https://download.racket-lang.org/releases/8.6/catalog/
Resolving "asd" via https://pkgs.racket-lang.org
Resolving "asd" via https://planet-compats.racket-lang.org
; main.rkt install: cannot find package on catalogs
;   package: asd
```

# 2. Using the `#:catalog` keyword in `pkg-install-command` does not change the catalogs
```
(pkg-install-command "asd" #:catalog "https://bing.com")
```
Result:
```
Resolving "asd" via https://download.racket-lang.org/releases/8.6/catalog/
Resolving "asd" via https://pkgs.racket-lang.org
Resolving "asd" via https://planet-compats.racket-lang.org
; main.rkt install: cannot find package on catalogs
;   package: asd
```

**Comment by mflatt** [comment]
Thanks for the report!

**Comment by xgqt** [comment]
Thanks a lot for the fix.

On September 5, 2022 4:49:47 AM GMT+02:00, Matthew Flatt ***@***.***> wrote:
>Closed #4427 as completed via 4e69f95e37153397abeeadfb8725a379a1ae3a65.
>

-- 
Have a great day!
~ Maciej XGQT Barć
